obs_status.yml

```diff
@@ -0,0 +1,2147 @@
+  mp_starcat_time: 2015:022:03:00:16.542
+  status: 1
+- agasc_id:
+  - 444743456
+  comments: Bright star. Almost never acquired. Set mag to 5.0
+  mp_starcat_time: 2015:024:19:18:39.261
+  status: 1
+- agasc_id:
+  - 444743456
+  comments: Bright star. Almost never acquired. Set mag to 5.0
+  mp_starcat_time: 2015:025:03:12:57.957
+  status: 1
+- agasc_id:
+  - 444743456
+  comments: Bright star. Almost never acquired. Set mag to 5.0
+  mp_starcat_time: 2015:025:05:06:07.571
+  status: 1
+- agasc_id:
+  - 111807640
+  comments: Faint star. Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2015:032:15:55:37.026
+  status: 1
+- agasc_id:
+  - 288753656
+  comments: Faint star. Tracking affected by imposter. Set magnitude to MAXMAG
+  mp_starcat_time: 2015:125:13:11:24.129
+  status: 1
+- agasc_id:
+  - 881460168
+  comments: Faint star. Affected by imposters
+  mp_starcat_time: 2015:130:19:50:00.443
+  status: 1
+- agasc_id:
+  - 1092505568
+  comments: Faint star. Affected by imposters
+  mp_starcat_time: 2015:159:15:26:54.646
+  status: 1
+- agasc_id:
+  - 262408104
+  comments: Short run. Magnitude seems ok
+  mp_starcat_time: 2015:201:22:05:03.421
+  status: 0
+- agasc_id:
+  - 653529952
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2015:212:20:00:08.125
+  status: 1
+- agasc_id:
+  - 821318496
+  comments: Faint star. Almost never tracked. Set mag to MAXMAG
+  mp_starcat_time: 2015:220:11:07:04.609
+  status: 1
+- agasc_id:
+  - 500706352
+  comments: Failed to track but there are snapshots
+  mp_starcat_time: 2015:231:00:04:31.256
+  status: 0
+- agasc_id:
+  - 144714600
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2015:233:22:11:26.142
+  status: 1
+- agasc_id:
+  - 144714600
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2015:234:03:22:43.521
+  status: 1
+- agasc_id:
+  - 376724944
+  comments: Mag 6.5 spoiler 45 arcsec away
+  mp_starcat_time: 2015:331:14:10:26.457
+  status: 1
+- agasc_id:
+  - 195167088
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2015:355:17:05:45.600
+  status: 1
+- agasc_id:
+  - 615257632
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2016:021:00:11:54.978
+  status: 1
+- agasc_id:
+  - 643702120
+  comments: Sparse periods of track at 12.5 mag; set mag to MAXMAG
+  mp_starcat_time: 2016:027:20:27:48.432
+  status: 1
+- agasc_id:
+  - 417611952
+  comments: Bright star. Fails to track. Set mag to 5.0
+  mp_starcat_time: 2016:062:05:39:12.185
+  status: 1
+- agasc_id:
+  - 118096888
+  comments: Faint star. Magnitude is good
+  mp_starcat_time: 2016:127:20:16:33.087
+  status: 0
+- agasc_id:
+  - 458500128
+  comments: Unexplained variability
+  mp_starcat_time: 2016:136:08:42:06.470
+  status: 0
+- agasc_id:
+  - 118359864
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2016:141:00:48:34.306
+  status: 1
+- agasc_id:
+  - 397673760
+  comments: UNKNOWN: SCS 107 at start of observation
```

Comment:
Obsid 18625 went to BSH but sure, SCS107 was probably in there.

Reply:
I wrote "UNKNOWN" because I could not figure out what happened, just that there was the SCS107.

Reply:
so the question is what do you suggest should be the comment?

Reply:
That's really a question about how we will use the comments.